sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -80,6 +80,58 @@ class StreamingAggregationSuite extends StateStoreMetricsTest with Assertions {
     }
   }
 
+  testWithAllStateVersions("prune results by localtimestamp, complete mode") {
```

Comment:
do we have a similar test for the old timestamp type?

Reply:
Yes.

Reply:
can you post the link? so that people can just compare the two tests and save code review time.

Reply:
https://github.com/apache/spark/blob/e8c17e589fd14866aff155eb1c96bf3481a742cd/sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala#L409